### ISSUE TITLE
Does latex tools support jumping around via a table of contents (like AUCTeX?)

### ISSUE THREAD

**Issue by jsinglet** [msiniscalchi/atom-latextools#91]
If not, I'd love to see this feature.

**Comment by pierregodard** [comment]
i don't think so, but [latex-friend](https://atom.io/packages/latex-friend) offers a TOC pane. it doesn't work very well will the pdf view inside Atom in my experience, and i also don't see a way to have the pane on the left instead of the right, but hopefully that could maybe already be something for you!

**Comment by msiniscalchi** [comment]
Or, you can use Atom's built-in Ctrl-R (Cmd-R on Mac) It will list sections
in a panel that you can use to navigate.

If this answers your question, please close.

On Mon, Jul 11, 2016 at 3:02 PM, pierregodard <notifications@github.com>
wrote:

> i don't think so, but latex-friend <https://atom.io/packages/latex-friend>
> offers a TOC pane. it doesn't work very well will the pdf view inside Atom
> in my experience, and i also don't see a way to have the pane on the left
> instead of the right, but hopefully that could maybe already be something
> for you!
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/msiniscalchi/atom-latextools/issues/91#issuecomment-231727671>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAo1BcnXtPSDELb5eCH-8PYC2864ZryNks5qUj8AgaJpZM4IatY->
> .
>



-- 
Marciano Siniscalchi
Economics Department, Northwestern University
http://faculty.wcas.northwestern.edu/~msi661

**Comment by maaaaadn** [comment]
the atom-latex package works pretty fine out of the box using latexmk and synctex. In the pdf you can easily jump around in the TOC and a simple ctrl+click leads you there.. that works fine for me. But still the auctex function is better.
